packages/paste-website/src/pages/components/popover/index.mdx

```diff
@@ -141,6 +146,22 @@ available in the [props description here](#popover-container).
   {rightExample}
 </LivePreview>
 
+### Badge Popover
+
+By default, the Popover is triggered by a Button. To use a Badge instead, add the `as="badge"` prop to the PopoverButton.
```

Comment:
You might want to say here (because we do a really bad job on the default example), that from this point the `PopoverButton` in all intense and purposes is now a `Badge` and all `Badge` Props are now valid. Link to the Badge docs. 

Maybe also include the 4 variants on badge in the example below? To hint at what you can do?

Might need to do that for the default button example above too honeslty to round out the docs.

Reply:
ahh good point! Update the docs

https://deploy-preview-2089--paste-docs.netlify.app/components/popover#modifying-the-popoverbutton